myia/infer/inferrers.py

```diff
@@ -50,10 +50,13 @@ def getattr_inferrer(node, args, unif):
     assert key_node.is_constant(str)
     obj = yield Require(obj_node)
     key = key_node.value
-    result = getattr(obj.tracks.interface, key)
+    interface = obj.tracks.interface
+    result = getattr(interface, key)
     if isinstance(result, (types.MethodType, types.WrapperDescriptorType)):
         ct = Constant(result)
         new_node = node.graph.apply(basics.partial, ct, obj_node)
+    elif isinstance(interface, types.ModuleType) and callable(result):
+        new_node = Constant(result)
```

Comment:
I feel this may not work very well if result is a callable class. Add `types.FunctionType` to the previous isinstance call and I think that should do the trick.